packages/core-data/src/reducer.js

```diff
@@ -411,8 +433,9 @@ export const entities = ( state = {}, action ) => {
 /**
  * @typedef {Object} UndoStateMeta
  *
- * @property {number} offset          Where in the undo stack we are.
- * @property {Object} [flattenedUndo] Flattened form of undo stack.
+ * @property {number} list   The undo stack.
+ * @property {number} offset Where in the undo stack we are.
+ * @property {Object} cache  Cache of unpersisted transient edits.
```

Comment:
What exactly is this cache?

Reply:
Sounds like that's just unpersisted edits, not really cache?

Reply:
It's a cache of "edits" that were not added to the "undo" stack yet.